bouvet-deler/databricks_lambda/databricks_lambda_case1.py

```diff
@@ -0,0 +1,72 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Using Lambda in Databricks - Case 1
+# MAGIC 
+# MAGIC This notebooks gives a simple example of using lambda in Databricks. For more information see this article (add link).
+# MAGIC 
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet1
+from pyspark.sql import Row
```

Comment:
can you put some air in here

Reply:
I think it looks better as is. Adding a black line makes it harder to read imo.